models/aeif_cond_beta_multisynapse.cpp

```diff
@@ -656,44 +686,37 @@ aeif_cond_beta_multisynapse_dynamics( double,
   // good compiler will optimize the verbosity away ...
 
   // shorthand for state variables
-  const double& V = y[ S::V_M ];
+  const double& V = std::min( y[ S::V_M ], node.P_.V_peak_ ); // bound V
   const double& w = y[ S::W ];
 
   double I_syn = 0.0;
 
```

Comment:
I would remove this empty line to tie the initialization more directly to the loop.